### PR TITLE
[delegate-registry-v2] Add optional v1 support

### PR DESCRIPTION
**Pull request by manboy-eth**
Adds optional support for "delegate registry v1" delegations in the "delegate registry v2" strategy.

### CODE REVIEW

File: src/strategies/delegate-registry-v2/index.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ const DEFAULT_BACKEND_URL = 'https://delegate-registry-backend.vercel.app';
 type Params = {
   backendUrl: string;
   strategies: Strategy[];
+  delegationV1VChainIds?: number[]; // add this to include v1 delegations
```
Comment (ChaituVR) [inline src/strategies/delegate-registry-v2/index.ts:15]: We are not using this anywhere?

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Also PR should be coming from `gnosis` github as they are the author, can fork this  from there? or better to create a new strategy

File: src/strategies/delegate-registry-v2/index.ts (hunk 1 above)
Comment (manboy-eth) [reply src/strategies/delegate-registry-v2/index.ts:15]: It is just forwarded to the backend in line 47 and used there.

Comment (ChaituVR) [reply src/strategies/delegate-registry-v2/index.ts:15]: Thanks @manboy-eth missed it

**Comment by ChaituVR** [comment]
I see you are original author of the strategy

**Comment by ChaituVR** [comment]
@manboy-eth Test cases are failing can take a look?
image

**Comment by manboy-eth** [comment]
Closing since I opened a PR from Gnosis: https://github.com/snapshot-labs/snapshot-strategies/pull/1353

I will also address the test error in that PR

**Pull request closed without merging**